relay-dynamic-config/src/global.rs

```diff
@@ -34,13 +40,27 @@ impl GlobalConfig {
     }
 }
 
+/// All options passed down from Sentry to Relay.
+#[derive(Default, Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(default, rename_all = "camelCase")]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct Options {
+    // Example:
+    // ```rs
+    // #[serde(default, rename = "relay.some-option.name")]
+    // pub some_option: Vec<u32>,
+    // ```
+    /// All other unknown options.
+    #[serde(flatten)]
+    other: HashMap<String, Value>,
```

Comment:
nit: I assume this `other` is for backward compatibility -- does using `RawValue` avoid deserializing the options we don't need? Haven't checked how serde deals with this exactly.